app/assets/javascripts/discourse/widgets/search-menu-results.js.es6

```diff
@@ -46,7 +47,8 @@ function postResult(result, link, term) {
 }
 
 createSearchResult('user', 'path', function(u) {
-  return [ avatarImg('small', { template: u.avatar_template, username: u.username }), ' ', u.username ];
+  console.log(u);
+  return [ avatarImg('small', { template: u.avatar_template, username: u.username }), ' ', h('span.user-results', h('b', u.username)), ' ',  h('span.user-results', u.name ? u.name : '') ];
```

Comment:
Now you have a different case where `name` might be not available. For example, the setting `enable_name` is disabled.